src/App.jsx

```diff
@@ -14,57 +15,47 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
-
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
-
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Garlic
-          </td>
-        </tr>
-      </tbody>
-    </table>
-  </main>
-);
+export const App = () => {
+
+  const [selectedGood, setSelectedGood] = useState('Jam')
+
+  return (
+    <main className="section container">
+      {selectedGood === '' ? <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+        : <h1 className="title is-flex is-align-items-center">
+          {selectedGood} is selected
+          <button data-cy="ClearButton" type="button" className="delete ml-3" onClick={() => setSelectedGood('')} />
+        </h1>
+      }
+
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr data-cy="Good" className={good === selectedGood && 'has-background-success-light'}>
+              {good === selectedGood ?
+                <td>
+                  <button
+                    data-cy="RemoveButton"
+                    type="button"
+                    className="button is-info"
+                    onClick={() => good === selectedGood ? setSelectedGood('') : setSelectedGood(good)}
```

Comment:
The condition `good === selectedGood ? setSelectedGood('') : setSelectedGood(good)` is redundant because the `good` will always be equal to `selectedGood` in this context. You can simplify this to `setSelectedGood('')`.